checkov/common/util/config_utils.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from pathlib import Path
+
+
+def config_file_paths(dir_path):
+    return [os.path.join(dir_path, '.checkov.yaml'), os.path.join(dir_path, '.checkov.yml')]
+
+
+def get_default_config_paths(argv):
+    """
+    Checkov looks for .checkov.yml or .checkov.yaml file in the directory (--directory) against which it is run.
```

Comment:
What about checking the current working directory? I know we talked about this, but I can't remember if we decided it made sense or not.

Reply:
Hey @mikeurbanski1, I was thinking about this use case some more. CWD might be handy for IDE extensions. Thoughts?

Reply:
Just closing the loop on this - Kartik will add CWD. Priority will be:

--config-file arg > --directory > CWD > home